Trappist/src/Promact.Trappist.Core/Controllers/CategoryController.cs

```diff
@@ -50,14 +56,15 @@ public IActionResult CatagoryAdd([FromBody] Category category)
         /// <param name="Id">Id is the primary key of Category Model</param>
         /// <param name="catagory">Object of  class Category</param>
         /// <returns>object of the class if key found or else it will return Bad request</returns>
+        
         [HttpPut("{id}")]
-        public IActionResult CategoryEdit(int Id, [FromBody] Category category)
+        public async Task<IActionResult> CategoryEditAsync(int Id, [FromBody] Category category)
```

Comment:
async suffix will not be there in controller action. and please rename it to Update Category